styles/header.css

```diff
@@ -32,23 +41,42 @@
   text-indent: 100%;
   white-space: nowrap;
   width: 250px;
+
+  &.polychrome {
+    background-color: transparent;
+    mask-image: none;
+    background-image: url("StanfordLibraries-logo-poly.svg");
+    background-repeat: no-repeat;
+    background-position: 0 center;
+  }
 }
 
 /* The .navbars selector is because adding .navbar to the nav in the masthead
  * causes popper to incorrectly position the dropdowns offscreen, so we need
  * to target that navigation with a different selector. */
-.bg-dark.navbar,
-.bg-dark .navbar,
-.bg-dark .navbars {
-  --bs-navbar-color: white;
-  --bs-navbar-active-color: var(--bs-navbar-color);
-  --bs-navbar-hover-color: rgb(255 255 255 / 100%);
-  --bs-nav-link-color: var(--bs-navbar-color);
-  --bs-navbar-toggler-icon-bg: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 17 17'%3e%3cpath stroke='rgba%28255, 255, 255, 1%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='1' d='M2.146 2.854a.5.5 0 1 1 .708-.708L8 7.293l5.146-5.147a.5.5 0 0 1 .708.708L8.707 8l5.147 5.146a.5.5 0 0 1-.708.708L8 8.707l-5.146 5.147a.5.5 0 0 1-.708-.708L7.293 8z' /%3e%3c/svg%3e");
-  --bs-navbar-toggler-border-color: rgb(255 255 255 / 10%);
-
-  .collapsed {
-    --bs-navbar-toggler-icon-bg: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba%28255, 255, 255, 1%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+.navbar,
+.navbars,
+.masthead {
+  --bs-navbar-color: var(--stanford-black);
```

Comment:
It doesn't seem appropriate to apply these variables to `.masthead` as they are only used in `.navbar`, right?

Reply:
The link-related ones were, but that was a small-ish chunk of this style block. Changed those to be part of `.masthead`'s rules later on.